pkg/kudoctl/cmd/install/install.go

```diff
@@ -248,18 +274,7 @@ func installSingleInstanceToCluster(name, previous, path string, gc *github.Clie
 	if vars.Parameter != nil {
 		p := make(map[string]string)
 		for _, a := range vars.Parameter {
-			// Using similar to CSV "," as the delimiter for now
-			// Split just after the first delimiter to support e.g. zk-zk-0.zk-hs:2181,zk-zk-1.zk-hs:2181 as value
-			s := strings.SplitN(a, ",", 2)
-			if len(s) < 2 {
-				return fmt.Errorf("parameter not set: %+v", s)
-			}
-			if s[0] == "" {
-				return fmt.Errorf("parameter can not be empty: %+v", s)
-			}
-			if s[1] == "" {
-				return fmt.Errorf("parameter value can not be empty: %+v", s)
-			}
+			s := strings.SplitN(a, "=", 2)
```

Comment:
Not required, but consider a Parameter struct here that we can re-use between the framework development SDK and the instance.

Reply:
Talked offline and decided to cover this in a followup PR